src/test/java/com/hocs/test/glue/LoginStepDefs.java

```diff
@@ -216,4 +238,17 @@ private void navigateToHocs() {
         }
         driver.get(baseUrl);
     }
+
+    private void navigateToManagementUI() {
+        String env = System.getProperty("environment");
+        String baseUrl = "";
+
+        if (env == null) {
+            System.out.println("Environment parameter not set. Defaulting to 'QA'");
+            baseUrl = Environments.MANAGEMENTUIQA.getEnvironmentURL();
+        } else {
+            baseUrl = Environments.MANAGEMENTUIDEV.getEnvironmentURL();
+        }
+        driver.get(baseUrl);
+    }
```

Comment:
Should this not be a switch statement so that environments that dont yet have a management UI page default to a pendingStep error message?

Reply:
You might be right. I wrote it this way because that's how HOCS currently works (you can see this in the stepdef) so I just replicated it for the Management UI